pkg/providers/docker/config/template-md.yaml

```diff
@@ -9,6 +9,18 @@ spec:
       joinConfiguration:
         nodeRegistration:
           criSocket: /var/run/containerd/containerd.sock
+{{- if .workerNodeGroupTaints }}
+          taints: {{ range .workerNodeGroupTaints}}
+            - key: {{ .Key }}
+              value: {{ .Value }}
+              effect: {{ .Effect }}
+{{- if .TimeAdded }}
+              timeAdded: {{ .TimeAdded }}
```

Comment:
Where in the code is this currently being added?

Reply:
So, Only NoExecute taints may come with TimeAdded. For other taint types, this field is invalid. Hence this check.

Reply:
I meant I don't see where in our code we are actually setting this value.

Reply:
It is part of the Taints struct.